From: Ankit Agrawal <ankita@nvidia.com>
Date: Sun, 13 Oct 2024 07:52:16 +0000
Subject: vfio/nvgrace-gpu: Add a new GH200 SKU to the devid table
Patch-mainline: v6.13-rc1
Git-commit: 12cd88a9116acf79416a39adcd8bb1337ae7cee1
References: jsc#PED-16989

NVIDIA is planning to productize a new Grace Hopper superchip
SKU with device ID 0x2348.

Add the SKU devid to nvgrace_gpu_vfio_pci_table.

Signed-off-by: Ankit Agrawal <ankita@nvidia.com>
Link: https://lore.kernel.org/r/20241013075216.19229-1-ankita@nvidia.com
Signed-off-by: Alex Williamson <alex.williamson@redhat.com>
Acked-by: Carlos López <clopez@suse.de>
---
 drivers/vfio/pci/nvgrace-gpu/main.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/vfio/pci/nvgrace-gpu/main.c
+++ b/drivers/vfio/pci/nvgrace-gpu/main.c
@@ -866,6 +866,8 @@ static const struct pci_device_id nvgrac
 	{ PCI_DRIVER_OVERRIDE_DEVICE_VFIO(PCI_VENDOR_ID_NVIDIA, 0x2342) },
 	/* GH200 480GB */
 	{ PCI_DRIVER_OVERRIDE_DEVICE_VFIO(PCI_VENDOR_ID_NVIDIA, 0x2345) },
+	/* GH200 SKU */
+	{ PCI_DRIVER_OVERRIDE_DEVICE_VFIO(PCI_VENDOR_ID_NVIDIA, 0x2348) },
 	{}
 };
 
